### PR TITLE
[Backport][ipa-4-10] ipatests: add Keycloak Bridge test

### PR DESCRIPTION
**Pull request by spoore1**
Adding test code for new bridge server (ipa-tuura) and Keycloak plugin.

Adding uninstall functions for create_keycloak.py so that the tests can
be run repeatedly.

Fixes: https://pagure.io/freeipa/issue/9227

### CODE REVIEW

File: ipatests/prci_definitions/nightly_ipa-4-10_previous.yaml [hunk 1]
```diff
@@ -1774,6 +1774,19 @@ jobs:
...
         topology: *master_2repl_1client
 
+  fedora-previous-ipa-4-10/test_sso:
+    requires: [fedora-previous-ipa-4-10/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-previous-ipa-4-10/build_url}'
+        update_packages: True
```
Comment (flo-renaud) [inline ipatests/prci_definitions/nightly_ipa-4-10_previous.yaml:1784]: lint detected an issue:
```
[File] nightly_ipa-4-10_previous [Jobs] 148
ERROR: In job 'fedora-previous-ipa-4-10/test_sso':
  'update_packages' field should be set to false or not defined
```

Comment (spoore1) [reply ipatests/prci_definitions/nightly_ipa-4-10_previous.yaml]: I missed that last night.  Fixed this issue earlier today.

**Comment by flo-renaud** [comment]
The failure in `test_commands` is a new issue, unrelated to this backport ([9244](https://pagure.io/freeipa/issue/9244) Nightly test failure in test_commands.py::TestIPACommand::test_ipa_cacert_manage_prune). A tentative fix is available in PR #6466.

**Comment by flo-renaud** [comment]
ipa-4-10:

* ac776987d30ecd3444a9b25f49a714fddc3c4232 ipatests: add Keycloak Bridge test
* db1d05176d8072b05fea179af2ac97caaeb65dd1 ipatests: add prci definitions for test_sso jobs

**Pull request closed without merging**